FashionConnect_ui/src/components/EditProfile/EditProfile.jsx

```diff
@@ -1,39 +1,45 @@
 import "./EditProfile.css";
 import React, { useState,useContext } from "react";
 import { Link, useNavigate } from "react-router-dom";
-
+import { UserContext } from '../../UserContext.jsx';
+// import { response } from "express";
 
 export default function EditProfile(){
     const [username, setUsername] = useState('');
     const [First_Name, setFirst_Name] = useState('');
     const [Last_Name, setLast_Name] = useState('');
-    
+    const { user, updateUser } = useContext(UserContext);
+    const navigate = useNavigate();
 
 
 
-    const navigate = useNavigate();
+    
     const handleProfileEdit = async (e) => {
       e.preventDefault();
 
       try {
-        const result = await fetch(`http://localhost:3000/users/profile`,{
+        const info = await fetch(`http://localhost:3000/users/profile`,{
```

Comment:
Let's also try to use clearer variable naming. This one can be update_profile_info for example.